Assets/MixedRealityToolkit/_Core/Extensions/VectorExtensions.cs

```diff
@@ -161,5 +161,86 @@ public static bool IsValidPosition(this Vector3 position)
             return !float.IsNaN(position.x) && !float.IsNaN(position.y) && !float.IsNaN(position.z) &&
                    !float.IsInfinity(position.x) && !float.IsInfinity(position.y) && !float.IsInfinity(position.z);
         }
+
+        /// <summary>
+        /// Get the relative mapping based on a source Vec3 and a radius for spherical mapping.
+        /// </summary>
+        /// <param name="source">The source <see cref="Vector3"/> to be mapped to sphere</param>
+        /// <param name="radius">This is a <see cref="float"/> for the radius of the sphere</param>
+        /// <returns></returns>
+        public static Vector3 SphericalMapping(Vector3 source, float radius)
+        {
+            float circ = 2f * Mathf.PI * radius;
+
+            Vector3 newPos = new Vector3(0f, 0f, radius);
+
+            float xAngle = (source.x / circ) * 360f;
+            float yAngle = -(source.y / circ) * 360f;
+
+            Quaternion rot = Quaternion.Euler(yAngle, xAngle, 0.0f);
+            newPos = rot * newPos;
+
+            return newPos;
+        }
+
+        /// <summary>
+        /// Get the relative mapping based on a source Vec3 and a radius for cylinder mapping.
+        /// </summary>
+        /// <param name="source">The source <see cref="Vector3"/> to be mapped to cylinder</param>
+        /// <param name="radius">This is a <see cref="float"/> for the radius of the cylinder</param>
+        /// <returns></returns>
+        public static Vector3 CylindricalMapping(Vector3 source, float radius)
+        {
+            float circ  = 2f * Mathf.PI * radius;
+            Vector3 newPos = new Vector3(0f, source.y, radius);
```

Comment:
same thing here. We probably don't need a new vector and can just update the source, then return it.

Reply:
I didn't bother to find this line in the previous PR, but this was probably also fixed there. (See my comment above)